src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -78,14 +77,15 @@ struct KeyValueViewModel: Comparable {
                 path: "client."
             )
         } else {
-            self.init(key: name, value: "?????")
+            self.init(key: param.name, value: "")
```

Comment:
Is this a situation we expect? If not, perhaps an assert would be wise here.

Reply:
Added an assert.